Makefile

```diff
@@ -72,11 +99,11 @@ check-migrations:  ## Check that all needed migrations exist
 	@echo "$(BOLD)Checking migrations$(RESET)"
 	@django-admin.py makemigrations --check --dry-run
 
-.PHONY: tests test
-test: tests  # we allow "test" and "tests"
+.PHONY: env-tests env-test
+test: env-tests  # we allow "test" and "tests"
```

Comment:
so, the rule name should be `env-test`

Reply:
@jmaupetit I reverted to the previous make file targets and introduced a USE_DOCKER environment variable so that we can use either Docker or Virtualenv with the same targets.

Reply:
@twidi normally everything should work like before with Virtualenvs. See here: https://github.com/openfun/marsha/blob/e3517769bf88cb900e4228a57c4ed76f0d5fd430/Makefile

You can test docker by running:
```
make bootstrap
make run
```
and visiting `localhost:8060`